helper/resource/error.go

```diff
@@ -0,0 +1,21 @@
+package resource
+
+type NotFoundError struct {
```

Comment:
It was an intention to actually make it consistent with what we already have in `master` - that is `CustomNameError`, not `ErrCustomName`.